packages/react/src/components/text-area/text-area.tsx

```diff
@@ -159,11 +160,17 @@ export const TextArea: VoidFunctionComponent<TextAreaProps> = ({
                 <Counter
                     data-testid="char-counter"
                     aria-live="polite"
-                    aria-hidden="true"
                     id={idCounter}
                     valid={inputValueLength <= maxLength}
                 >
-                    {t('characters', { current: inputValueLength, max: maxLength })}
+                    {inputValueLength}
+                    {' '}
+                    <span aria-hidden="true">/</span>
+                    <ScreenReaderOnlyText label="of" />
+                    {' '}
+                    {t('characters', { max: maxLength })}
+                    {' '}
+                    <ScreenReaderOnlyText label="available" />
```

Comment:
@LarryMatte @carlafranca

I was thinking if we could do something like this instead?  It's just so awkward and ugly to mix both in the same sentence.

```
<span aria-live="polite" aria-label="12 of 30 characters available">
  12 / 30 characters
</span>
```
or
```
<span aria-live="polite">
  <ScreenReaderOnlyText label="12 of 30 characters available" />
  <span aria-hidden="true">12 / 30 characters</span>
</span>
```

BTW I had this idea just immediately after doing my PR

Reply:
1st proposition:
You cannot use an `aria-label` on a `span` element that does not have an associated ARIA role, making this option unfeasible.

2nd proposition:
As of now, I have nothing against it.

Reply:
Done with solution 2